src/redis/stream.h

```diff
@@ -170,7 +170,7 @@ robj *createObjectFromStreamID(streamID *id);
 int streamAppendItem(stream *s, robj **argv, int64_t numfields, streamID *added_id, streamID *use_id, int seq_given);
 int streamDeleteItem(stream *s, streamID *id);
 void streamGetEdgeID(stream *s, int first, int skip_tombstones, streamID *edge_id);
-long long streamEstimateDistanceFromFirstEverEntry(stream *s, streamID *id);
+// long long streamEstimateDistanceFromFirstEverEntry(stream *s, streamID *id);
```

Comment:
@romange Will it be okay to use this method implementation from the original redis source code ? I can see it as commented out right now.
I have replicated it in `stream_family.cc` for now.

Reply:
it's totally fine reusing the existing functions under src/redis instead of copying code to stream_family if it's possible.

Reply:
looked at it again. Please uncomment it in t_stream.cc. 

@kostasrim  I am taking ownership of this review. @rounaknandanwar  Please tell me if you want to proceed.

Reply:
@romange shall I still go over it or?